src/main/java/seedu/duke/command/AddNoteCommand.java

```diff
@@ -48,17 +52,31 @@ public String execute() {
         }
 
         // Get Content
+        try {
+            if (storageManager.noteExists(note)) {
+                content = storageManager.getNoteContent(note);
+            }
+        } catch (SystemException exception) {
+            return Formatter.formatString(exception.getMessage());
+        }
+
```

Comment:
Why is there a getNoteContent here since it is adding a new note? I believe the line below that @Nazryl added is to pass his JUnit test.

Reply:
Oh I added it bc otherwise, when I use the add command function, it asks for the user input for the content of the note. So when loading the files I dont want to ask for the user input, therefore check if a content file exists (aka are we loading the data) and accordingly see where to get the content from